homeassistant/components/omnilogic/sensor.py

```diff
@@ -248,7 +251,7 @@ def native_value(self):
         return orp_state
 
 
-SENSOR_TYPES = {
+SENSOR_TYPES: dict[tuple[int, str], list[dict[str, Any]]] = {
```

Comment:
This component would benefit from refactoring to EntityDescription, but that's a bigger job and I think it's still useful to first take it out of `mypy` ignore list.